astropy/modeling/powerlaws.py

```diff
@@ -492,6 +491,14 @@ class Schechter1D(Fittable1DModel):
     power-law form of the function cuts off. Note that the default value for
     ``l_star`` is in units of solar luminosities, in erg/s, to avoid overflow
     errors.
+
+    References
+    ----------
+    
+    .. _ref-schechter1976:
```

Comment:
Indeed we are not yet uniform with the format, we need to fix it for the Voigt1D, too. Sorry for the nitpicking.

I suspect @crawfordsm has something like this in mind: https://github.com/astropy/astropy/blob/master/astropy/modeling/functional_models.py#L745